GOVERNANCE.md

```diff
@@ -0,0 +1,195 @@
+
+### Maintainer
+
+Maintainers are those with a history of consistent contributions, including but not limited to pull requests, project management, or support. These privileges include:
+
+- Push access to some of the repositories of the [Biome GitHub org][gh-org]
+- `Maintainer` status on the [Biome Discord server][discord]
+- Ability to [vote](#voting) on project decisions
+- Ability to [vote](#maintainer-nomination) on new maintainers
+
+#### Maintainer nomination
+
+Maintainers may either be nominated by another [maintainer](#maintainer), [Core contributor](#core-contributor), [lead](#lead).
+
+When nominating a new maintainer, the person has to create a new thread in the `#maintainers` channel, provide a link to the history of the person's contributions to the project and a brief explanation of why the person should join as [maintainer](#maintainer).
+
+`#maintainers` is a private channel available only to people who have the `maintainer` role.
+
+The voting period will follow the [voting rules](#voting-rules). An individual becomes a [maintainer](#maintainer) if:
+- reaches more than 50% of positive votes among the people who have voting rights;
+- if the voting session ends with 50%, the Lead maintainers will take the decision;
+
+A maintainer can be self-nominated. They will have to message a [lead maintainer](#lead) privately with links to their contributions to the project. Then, the lead can:
+- reject the nomination in place, provide reasons why and possibly some suggestions for the future;
+- initiate a voting session among the maintainers;
+
+Upon termination of a nomination, the thread will be deleted, and a [core contributor](#core-contributor) will reach out to the nominee and ask them if they want to accept the title.
+
+In the event of a rejection, the nominated person will be privately given the requirements they have not met. Details of the discussion, such as the names of those who objected the nomination, will not be disclosed.
+
+
+### Voting Rules
+
+- Members may abstain from a vote.
+- Members who do not vote within seven days will automatically abstain.
+- Leads may reduce the seven days automatic abstain for urgent decisions.
+- Leads reserve the right to veto approval with a publicly disclosed reason.
+
+### Ownership
+
+One or more people can own certain parts of the codebase. This process is informal, and inclusion could result from substantial contributions or delegation by other members. A maintainer's responsibility is to identify the relevant owners and ensure there's an understanding when it comes to code review.
+
+
+### Inactivity
+
+There are no expectations around activity once someone becomes a core contributor or maintainer. Inactive core contributors or maintainers may have voting rights removed; however, will always retain their status. A core contributor or maintainer may request their voting rights back upon sufficient activity.
```

Comment:
One of the hardest lessons I learned from the io.js Governance Structure that ended up as the Node.js Governance Structure was that the decision to _never_ remove inactive members was perhaps one of the biggest barriers to progress. Eventually, we introduced an emeritus system that IMO has been extremely helpful in distinguishing who is an active contributor vs. who once was but is not longer active.

The Emeritus system also helps ensure a more secure environment, since inactive accounts don't have push permissions, while still allowing for recognition and an easy way to start working again (simply ask to be re-added to the project from emeritus status, and you will be).

To be in good conscience, I feel obligated to at least leave a comment urging you to consider a way to build-in a system to step-down or off-board people who are inactive after having worked in such a system without an escape hatch like Emeritus for so long.